io.edcd.EDMarketConnector.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: io.edcd.EDMarketConnector
```

Comment:
does `edcd.io` really exist and use for that software?

Reply:
We wondered if you might object to that.

1. The webpage is served from github at https://edcd.github.io/
2. As noted elsewhere the home of this application is https://github.com/EDCD/EDMarketConnector/  with the documentation being at https://github.com/EDCD/EDMarketConnector/wiki .

At this time the host EDMarketConnector.edcd.io doesn't even exist, and we'd only be creating it for this purpose if you insist on it.

But in terms of app-id `io.edcd.EDMarketConnector` seems correct and is definitely going to be unique.

Reply:
> We wondered if you might object to that.
> 
>     1. The webpage is served from github at https://edcd.github.io/
> 
>     2. As noted elsewhere the home of this application is https://github.com/EDCD/EDMarketConnector/  with the documentation being at https://github.com/EDCD/EDMarketConnector/wiki .


Then the app-id as per the documentation should be `EDMarketConnector.edcd.github.io`.

> But in terms of app-id `io.edcd.EDMarketConnector` seems correct and is definitely going to be unique.

it's not a uuid. There are rules for a reason.

Reply:
Yeah, it should either use GitHub's domain for user-generated content, or I'd recommend registering this domain. This is critical for upcoming first-party verification feature.

Reply:
I should have mentioned that, despite the active webpage being edcd.github.io, the domain edcd.io *does* exist, and is used by this non-profit organisation.  EDCD is "Elite Dangerous Community Developers".  An example of the domain's usage is eddn.edcd.io (the live version of the https://github.com/EDCD/EDDN project).

I can get edmarketconnector.edcd.io created (and likely then serving a temporary redirect to the github page) if it's absolutely necessary.

Reply:
If it's a registered domain, and you either control it or are related to that organization, it's fine to keep it as is.

Reply:
To ensure no issues going forwards (i.e. someone reviewing a future PR for this) I've requested the hostname be created.  Once it is I'll configure the relevant web server to have the temporary redirect to the github page.

Reply:
https://edmarketconnector.edcd.io/ is now active, redirecting to the wiki page.